src/request/messages/ILogMessage.ts

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+export interface ILogMessage {
+    level: string;
+    message: string;
+}
```

Comment:
Will be interesting to see what we end up doing with contracts like this as they are more of a mixin rather than a single object.